### PR TITLE
Feature NEX 1211 - Use language endpoint shared stimulus

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to**

https://oat-sa.atlassian.net/browse/NEX-1211

**Description**

Remove mocked data language list and load data from the endpoint 

`GET tao/Languages/index`

**AC**

User can select any platform language and save it in the Assets Authoring page

### CODE REVIEW

File: views/js/qtiCreator/helper/sharedStimulusLoader.js [hunk 1]
```diff
@@ -31,58 +32,40 @@ define([
...
 
+    const languagesUrl = urlUtil.route('index', 'Languages', 'tao');
+
     const creatorLoader = {
         loadSharedStimulus(config, callback) {
 
             if (config.id) {
-                request(config.assetDataUrl, { id : config.id })
-                    .then(function(data){
+                request(languagesUrl)
```
Comment (bziondik) [inline views/js/qtiCreator/helper/sharedStimulusLoader.js:41]: These 2 requests are independent, we can load them in parallel

Comment (habemuscode) [reply views/js/qtiCreator/helper/sharedStimulusLoader.js]: done

File: views/js/qtiCreator/helper/sharedStimulusLoader.js [hunk 2]
```diff
@@ -31,58 +32,37 @@ define([
...
+                        loadedItem.setNamespaces(namespaces);
+                        loadedItem.setSchemaLocations(qtiSchemaLocation);
 
-                            //add languages list to the item
-                            if (languagesList) {
-                                loadedItem.data('languagesList', languagesList);
-                            }
+                        //add languages list to the item
+                        if (languagesList) {
+                            loadedItem.data('languagesList', values[0].data);
```
Comment (bziondik) [inline views/js/qtiCreator/helper/sharedStimulusLoader.js:60]:
```suggestion
                            loadedItem.data('languagesList', values[0]);
```

File: views/js/qtiCreator/helper/sharedStimulusLoader.js [hunk 3]
```diff
@@ -31,58 +32,37 @@ define([
...
-                            'nl-BE': 'Flemish',
-                            'nl-NL': 'Dutch',
-                            'pt-PT': 'Portuguese',
-                            'ru-RU': 'Russian',
-                            'sv-SE': 'Swedish',
-                            'uk-UA': 'Ukrainian',
-                            'zh-CN': 'Simplified Chinese from China',
-                            'zh-TW': 'Traditional Chinese from Taiwan'
-                        };
+                const languagesList = request(languagesUrl)
```
Comment (bziondik) [inline views/js/qtiCreator/helper/sharedStimulusLoader.js:41]:
```suggestion
                const languagesList = request(languagesUrl);
```

File: views/js/qtiCreator/helper/sharedStimulusLoader.js [hunk 4]
```diff
@@ -31,58 +32,37 @@ define([
...
+                const assetData = request(config.assetDataUrl, { id : config.id })
```
Comment (bziondik) [inline views/js/qtiCreator/helper/sharedStimulusLoader.js:42]:
```suggestion
                const assetData = request(config.assetDataUrl, { id : config.id });
```

**Pull request merged**